index.html

```diff
@@ -3,24 +3,34 @@
     <head>
         <title>Kaleb Mattison's Portfolio</title>
         <meta charset="UTF-8" />
-        <link href= "C:\Users\kjmat\Documents\portfolio-site\index.css" rel= "stylesheet" type= "text/css" />
+        <link href=index.css rel= "stylesheet" type= "text/css" />
     </head>
     <body>
         <h1>Kaleb Mattison</h1>
-        <img src="C:\Users\kjmat\OneDrive\Desktop\Kaleb's Docs\Photos\profile pic 1.jpg" width="260" height="200" />
-        <div>I love combining adventure with new tech!</div>
-        <div>What adventure would you like to take today?</div>
+        <img src="https://lh3.googleusercontent.com/ZykwrjFt3cedUVi-4DkSCH0wX8Zu4AK_yzwKe-jiGiC4kSJ_6mIvps5UqJ7rYjGtCf8r1odQxzpAYZ8NJoIn7v_VD57Oxr_UZ5lWMJpg3GU2t9F56nWgx-L13FCMBG5pietKtuwCGL9FX7WfcthbAlCtNyCWoGx3RnY-8ViwH5kPD4rfnKaILFj2QRUsBPG-2iUCxweErpPGRNOCRXZIb79UudoZAVhAgMGBD0fZXhH4vCkWfC4YLnVTRjUAE2B7mx-bDzPpWnvu98MAAKFZFVgyu3J_CC8ivVxwxLGQrIH2EctAb6wWVeOU9dhvsOJdGQnupQsU_wxfEAQzfmnTbdZQZ-5KLTEBz3THQCItx3GL_TuzlGfGbmu7dWBfixZrDOlCYAZtRUNvKBlPUYvhpBK9NheoN6z4HhXHyYCdpW8EfjvaZYWW3JkClezIoX_cXm-NDeQGc7KzZJQU2AHnuRvUe2lRYk5wlbdJVR9WMxW7g2rCJy-h4zE1r27-dhE0zwSDCwGwT-kcwRRCWAsgsC2U8KRAFSataKlJ3kldj_K_F2UtCmcrj9POP3z0Ydoo8XaM2L3SBRpGEtXnRrF1PUNxIVI5pavHytwLhYqsZ1I0Zj7V9l-oTbm4dIBYIyyMyCCWanrMIV4zPksh9IN0cMNI2fn7cExAArE0qQFgPASlqoL5xTuqoc_q8Iqjdiy2PkPoXIxDkWVAXvP1ZfXX2L7g=w1255-h943-no?authuser=0"  width="260" height="200" />
+        <div class="text">
+            <p>I am a loving husband and father with a focus on my family.</p>
+            <p>I love going on family adventures such as biking, camping and hiking </p>
+            <p>and combining those adventures with new tech!</p>
+            <p>What adventure would you like to take today?</p>
+        </div>
+
         <div class="byclass"></div>
-        <ul>
-            <li>camping</li>
-            <li>biking</li>
-        </ul>
-        <p> <b>Words of Wisdom</b> </p>
-        <ul>
-            <li>"Time is the only thing you can waste but can't buy"</li>
-            <li>"If you are not willing to learn, no one can help you.
-                If you are determined to learn, no one can stop you."</li>            </li>
-        </ul>
-        <a rel="stylesheet" target="_blank" href="https://github.com/kjmatt91" >GitHub </a>
+        <div class="arrow-up"></div>
+        <div class="arrow-down"></div>
+        <div class="arrow-left"></div>
+        <div class="arrow-right"></div>
+        
+        <div class="quotes">
+            <p> <b>Words of Wisdom</b> </p>
+                <ul>
+                    <li>"Time is the only thing you can waste but can't buy"</li>
+                    <li>"If you are not willing to learn, no one can help you.
+                         If you are determined to learn, no one can stop you."</li>
+                </ul>
+        </div>
+        <div class="links">
+            <a rel="stylesheet" target="_blank" href="https://github.com/kjmatt91" >GitHub </a>
```

Comment:
why rel="stylesheet" here?

Reply:
Honestly... I have no idea... I may have seen something similar somewhere in the slides and just inserted it. Or I may have started typing in the wrong line... I don't have a solid answer for you on that.

Reply:
some reason this link takes over the whole page. so we can click on anything else.